conda/conda-build/conda_build_config.yaml

```diff
@@ -1,8 +1,6 @@
-# Conda only accepts strings as variant configs
-# so use an empty string "" to denote a falsey value
 gpu_enabled:
   - true
-  - 
+  - false
```

Comment:
cc @m3vaz is this conda issue no longer an issue?

Reply:
It's still an issue in the sense that you can't have true boolean variables. After seeing the problems this caused for you, I thought we could just use an explicit string for false.

Reply:
@bryevdv nope. Marcin insisted on the magic words. I've tried to mitigate the possible damage by defining a local variable in the jinja template. 
```
{% if gpu_enabled == "true" %}
  {% set gpu_enabled_bool = true %}
{% elif gpu_enabled == "false" %}
  {% set gpu_enabled_bool = false %}
{% else %}
  {# We need to have a default value for the initial pass over the recipe #}
  {% set gpu_enabled_bool = false %}
{% endif %}
```